wrapcheck/wrapcheck.go

```diff
@@ -273,6 +277,16 @@ func reportUnwrapped(
 	pkgGlobs []glob.Glob,
 ) {
 
+	if cfg.ReportInternalErrors {
+		// Check if the call is package-internal function
+		fnIdent, ok := call.Fun.(*ast.Ident)
+		if ok {
+			fnSig := pass.TypesInfo.ObjectOf(fnIdent).String()
```

Comment:
Also are we potentially missing a signature check here? This path appears to miss the ignore list. If so, could we add a test to check for the ignore list works on an internal function as well?